tree.go

```diff
@@ -1557,25 +1560,25 @@ func (n *InternalNode) collectNonHashedNodes(list []VerkleNode, paths [][]byte,
 }
 
 // unpack one compressed commitment from the list of batch-compressed commitments
-func (n *InternalNode) serializeInternalWithCompressedCommitment(compressedPointsIdxs map[VerkleNode]int, compressedPoints [][32]byte) ([]byte, error) {
-	serialized := make([]byte, nodeTypeSize+bitlistSize+SerializedPointCompressedSize)
-	bitlist := serialized[internalBitlistOffset:internalNodeChildrenOffset]
+func (n *InternalNode) serializeInternalWithUncompressedCommitment(pointsIdx map[VerkleNode]int, serializedPoints [][SerializedPointUncompressedSize]byte) ([]byte, error) {
```

Comment:
Also tried to rename variables to avoid referring to "compressed" or "uncompressed"; and use a more general "serialized". (If it's compressed or uncompressed, should be irrelevant)